### ISSUE TITLE
Rules in tsconfig.json don't seem to be respected

### ISSUE THREAD

**Issue by rsslldnphy** [mhartington/nvim-typescript#253]
# Problems summary

I have set `unusedLocals` and `unusedParameters` to `false` in my `tsconfig.json`, but `nvim-typescript` is still showing them as errors. I'm not sure if it's not finding my `tsconfig.json` or if it's just ignoring those specific rules.

Running `tsc` shows no errors.

## Expected

No errors should be shown for unused locals or parameters when these options are turned off in the `tsconfig.json`.

## Environment Information

 * terminal: iterm2
 * vim/nvim version: NVIM v0.3.8

## Provide a minimal vim rc with less than 50 lines (Required!)

```vim
set nocompatible
filetype plugin indent on
syntax on

call plug#begin('~/.config/nvim/vim-plug')

Plug 'HerringtonDarkholme/yats.vim', { 'for': 'typescript' }
Plug 'mhartington/nvim-typescript',  { 'do': './install.sh', 'for': 'typescript' }

call plug#end()
```

## The reproduce ways from neovim starting

1. open a typescript file
2. write a function with an unused parameter 

## Example project repo

https://github.com/rsslldnphy/nvim-typescript-issue

## Screen shot (if possible)

![image](https://user-images.githubusercontent.com/1231200/63640673-f2116400-c69a-11e9-9a6b-037b9b57f5af.png)

**Comment by mhartington** [comment]
Looked into this, seems it is just a "suggested" diagnostic report and not an actual error. 
For real errors, like when those settings are enabled, they should be highlighted with a red symbol.

<img width="870" alt="Screen Shot 2019-08-25 at 9 08 41 PM" src="https://user-images.githubusercontent.com/2835826/63658942-856e9600-c77c-11e9-81d4-0c2a8a787229.png">


This just typescript trying to provide some "helpful" feedback.

**Comment by rsslldnphy** [comment]
Thanks for looking into this - could you point me to where you discovered that? And is it possible to turn it off? The dots are the same colour for me regardless of whether the rule is turned on.

**Comment by rsslldnphy** [comment]
Ok i think i've found where this is handled in the plugin. Would you accept a PR that made showing suggestions optional?
